### ISSUE TITLE
关于 Log Loss

### ISSUE THREAD

**Issue by TangJiakai** [LaVieEnRose365/ReLLa#15]
请问好像不是所有 LLM（当然原论文是没问题的），对于其他LLM，比如 Instruct 系列，发现他们输出 yes 和 no 的 logits 相差挺大的，导致 LogLoss 比较高，这种有什么解决方法吗？

**Comment by LaVieEnRose365** [comment]
您好，我们猜测这可能与LLM本身的能力有关。我们论文附录中Table 5在兼容性实验时也测了mistral和falcon，都是用的Instruct系列，mistral与vicuna水平相当，但falcon就略差。

**Comment by TangJiakai** [comment]
感谢回复。论文这些都比较符合预期，但我实验用比如 LLaMA-Instruct，或者 Qwen-Instruct等，LogLoss 都会算出接近 10 这么高了

**Comment by LaVieEnRose365** [comment]
按理来说应该不会，这么高的LogLoss可能就是训崩了。您可以检查一下您的chat template格式是否与模型匹配，以及yes 和 no的位置在不同模型词表中的位置（compute_metrics里）也会不一样，这些都是需要根据模型定制的。

**Comment by TangJiakai** [comment]
我实验了，应该没错，而且我没训，直接用开源 LLM，yes 和 no 这个我知道的，我是可以确保实现应该没问题。
切换到 vicuna，或者原文的这些大模型，是不会出现这种问题。 我打印了 logit，就是因为模型对于 yes/no的 score 要 较高于 no/yes的 score，而 Rella 论文里这些，基本 yes 和 no 的 logit 差不了太多，所以说 logloss 不那么离谱。
而且我 auc 和 acc 的指标也还行。
